src/utils.cxx

```diff
@@ -154,102 +155,24 @@ std::string getCmdOption(char ** begin, char ** end, const std::string & option)
     return 0;
 }
 
-bool IsRelocate(const char * arg)
-{
-    return strcmp(arg, "relocate") == 0;
-}
+void StripPathAndExe(std::string &command) {
+    StripPath(command);
+    StripExe(command);
+};
 
-/**
- * Checks if an argument has already been defined on the command line
- */
-int redefinedArgCheck(const std::map<std::string, std::string> &args, const char * arg, const char * cli_name)
-{
-    if ( args.find(arg) != args.end()) {
-        std::cerr << "Invalid command line, too many values for argument: " << cli_name << "\n";
-        return 1;
-    }
-    return 0;
+void StripExe(std::string &command) {
+    // Normalize command to lowercase to avoid parsing issues
+    std::transform(command.begin(), command.end(), command.begin(),
+        [](unsigned char c){ return std::tolower(c); });
+    std::string::size_type loc = command.rfind(".exe");
```

Comment:
This should generally work, but I think it assumes that if you find `.exe` that is the end of the command, but could e.g. be `.exechusetts` (made up example). I recommend "refining" this in a follow up.

Reply:
Refined here, updated.